src/features/contracts/index.ts

```diff
@@ -135,6 +135,10 @@ const upsertEntries = async (
 	await Bluebird.map(
 		newData,
 		async (entry: any) => {
+			const entryQuery =
+				'contract' in entry && typeof entry.contract === 'object'
```

Comment:
I think you should change this to
```suggestion
				'contract' in entry && entry.contract != null
```
since we would still stringify it on entry if it's a string/number/etc, it's only null that would be left as-is

Reply:
Fixed in new commit